test/skimNtuple_HHbtag.cpp

```diff
@@ -267,13 +215,127 @@ int main (int argc, char** argv)
+	DYscale_MTT_vLowPt	= {0.87720949, 0.010000006, 0.010000025}; 
+	DYscale_MTT_LowPt  	= {1.2191486 , 0.010001064, 0.29051790 }; 
+	DYscale_MTT_Med1Pt	= {1.1816037 , 0.82760074 , 0.84809836 }; 
+	DYscale_MTT_Med2Pt	= {1.1579303 , 1.1240148  , 0.92974364 }; 
+	DYscale_MTT_HighPt	= {1.0469869 , 1.3690206  , 1.0024774  }; 
+	DYscale_MTT_vHighPt	= {0.80838089, 1.7465338  , 0.73211715 };
+  }
+  else if (PERIOD == "2017") {
+	DYscale_MTT_vLowPt  = {1.1092067 , 0.010000025, 0.010000002};
+	DYscale_MTT_LowPt   = {1.4229782 , 0.025083862, 0.64151569 };
+	DYscale_MTT_Med1Pt  = {1.3143450 , 1.0076030  , 0.96584965 };
+	DYscale_MTT_Med2Pt  = {1.2485879 , 1.2376612  , 1.0745893  };
+	DYscale_MTT_HighPt  = {1.1273438 , 1.5564765  , 1.0578688  };
+	DYscale_MTT_vHighPt = {0.87293424, 1.4469675  , 1.1665250  };
+  }
+  else if (PERIOD == "2016preVFP" or PERIOD == "2016postVFP") {
+	DYscale_MTT_vLowPt  = {1.1630144, 0.010000393, 0.010000000};
+	DYscale_MTT_LowPt   = {1.2194740, 1.1250249  , 1.0609708	};
+	DYscale_MTT_Med1Pt  = {1.2536864, 1.2376837  , 1.1901911	};
+	DYscale_MTT_Med2Pt  = {1.2763251, 1.2972053  , 1.2731480	};
+	DYscale_MTT_HighPt  = {1.2785250, 1.4578434  , 1.2241989	};
+	DYscale_MTT_vHighPt = {1.1649714, 1.6778047  , 1.1510545	};
+  }
+
+  std::array<float, 3> DYscale_MTT_vLowPt_err ;
+  std::array<float, 3> DYscale_MTT_LowPt_err  ;
+  std::array<float, 3> DYscale_MTT_Med1Pt_err ;
+  std::array<float, 3> DYscale_MTT_Med2Pt_err ;
+  std::array<float, 3> DYscale_MTT_HighPt_err ;
+  std::array<float, 3> DYscale_MTT_vHighPt_err;
+  if (PERIOD == "2018") {
+	DYscale_MTT_vLowPt_err  = {0.0014349486, 0.00085205781, 0.00094359815};
+	DYscale_MTT_LowPt_err	= {0.0013417125, 0.0044914533 , 0.025179988  };
+	DYscale_MTT_Med1Pt_err  = {0.0016262942, 0.021066553  , 0.025609469  };
+	DYscale_MTT_Med2Pt_err  = {0.0014769770, 0.018984114  , 0.019089746  };
+	DYscale_MTT_HighPt_err  = {0.0022691835, 0.035830965  , 0.027803721  };
+	DYscale_MTT_vHighPt_err = {0.0046517215, 0.11443309   , 0.064519398  };
+  }
+  else if (PERIOD == "2017") {
+	DYscale_MTT_vLowPt_err  = {0.0020977215, 0.0014144677, 0.0015785401};
+	DYscale_MTT_LowPt_err   = {0.0018789754, 0.028074073 , 0.033416855 };
+	DYscale_MTT_Med1Pt_err  = {0.0022982702, 0.028221735 , 0.032660541 };
+	DYscale_MTT_Med2Pt_err  = {0.0019811270, 0.024603319 , 0.024254387 };
+	DYscale_MTT_HighPt_err  = {0.0030012172, 0.046576904 , 0.035949714 };
+	DYscale_MTT_vHighPt_err = {0.0061768066, 0.15213682	, 0.091582069 };
+  }
+  else if (PERIOD == "2016preVFP" or PERIOD == "2016postVFP") {
+	DYscale_MTT_vLowPt_err  = {0.0037446320, 0.011371985, 0.0071346649};
+	DYscale_MTT_LowPt_err   = {0.0017404799, 0.023737485, 0.035623816 };
+	DYscale_MTT_Med1Pt_err  = {0.0027420531, 0.034357252, 0.043622931 };
+	DYscale_MTT_Med2Pt_err  = {0.0042018892, 0.055512921, 0.061654929 };
+	DYscale_MTT_HighPt_err  = {0.0045670912, 0.065499641, 0.064115483 };
+	DYscale_MTT_vHighPt_err = {0.0084286985, 0.17367880 , 0.13585326	};
+  }
+
```

Comment:
I think we will not use this scale factors with the DY NLO samples: the agreement in the MuMu channel is quite good for all the years as far as I remember.

Reply:
Given that this PR is meant to adapt the code to a single executable and not to change much in the code (except fro the muon selection that really is a small detail), I propose to remove the above in a near-future "clean-up" PR, where #312 will also be addressed.